### ISSUE TITLE
[A11Y] [GroupedList] - aria-required-children attribute is not set on the group header

### ISSUE THREAD

**Issue by pankajkhanzode** [OfficeDev/office-ui-fabric-react#1071]
### Bug Report

- __Package version(s)__: (1.7.2)
- __Browser and OS versions__: (all)

#### Describe the issue:

In GroupedList, when groups are added and each group contains elements - ARIA attribute when role="grid" is set - should have aria-required-children also set.
![image](https://cloud.githubusercontent.com/assets/11996243/23197258/1c786c88-f8e5-11e6-94a7-05cba17a7cad.png)

#### Actual behavior:

aria-required-children validation fails when role="grid" other validations passed:
![image](https://cloud.githubusercontent.com/assets/11996243/23197297/4b76bd1e-f8e5-11e6-85e4-3fcef847bc87.png)
[Done using Keros tool in VSTS org]

#### Expected behavior:

Set the aria-required-children on group header as suggested here: https://dequeuniversity.com/rules/axe/1.1/aria-required-children

### If applicable, please provide a codepen repro:

https://fabricreact.azurewebsites.net/fabric-react/master/#examples/groupedlist

**Comment by kristoferbrown** [comment]
I'm no longer getting aria-required-children errors when doing accessibility tests using aXe or the Chrome accessibility audit plugin.

@pankajkhanzode, if you are still seeing this issue please reopen this ticket with more details.
